code/index.html

```diff
@@ -1,30 +1,104 @@
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Simple Responsive Layout</title>
+        <link rel="stylesheet" href="dist/style.css">
+        
+    </head>
+    <body>
+        
+        <!-- Main Content -->
+        <main>
+            <!-- Header -->
+            <header class="header">                            
+                <p><a href="mailto:bergman.signe@gmail.com" class="square_btn">IN NEED OF A BABYSITTER? LOOK NO FURTHER!</a></p>
+            </header>
+
+            <!-- Hero -->
+            <section class="hero">
+                <picture class="hero-image">
+                    <source media="(min-width:1200px)" srcset="./assets/desktopsmall.png">
+                    <source media="(min-width:768px)" srcset="./assets/tabletsmall.png">
+                    <img src="./assets/mobilsmall.png" alt="Signe" style="width:auto;">
+                  </picture> 
+
+                <div class="hero-title">
+                    <h1>- SIGNE BERGMAN -</h1>
+            </section>
+
+            <div class="flex-container wrapper">
+
+                <!-- Content -->
+                <section class="content">
+            <!-- FAQ -->
+
+            <button class="accordion">WHO AM I?</button>
+            <div class="panel">
+            <p>Hi! I am turning 16 this april and i am currently in ninth grade at Skarpängsskolan, Täby. 
+                This fall I plan to start studying gymnasium level Economics.</p>
+                <p>I come from a big family and have two older brothers and five bonus siblings!</p>
+                <p>I have a lot of previous experience with children from my job as a gymnastics coach for Sollentungymnasterna.</p>
+            </div>
+    
+            <button class="accordion">WHAT CAN I DO?</button>
+            <div class="panel">
+            <p>I will give 100% of my attention to your kids.</p>
+            <p>Sitting will take place at your house, I will get there myself.</p>
+            </div>
+    
+            <button class="accordion">HOW?</button>
+            <div class="panel">
+            <p>Looking forward to hearing from you. Feel free to fill out the form. Tell me a bit about your family, what you would like help with, and I will get back to you soon.</p>
+            <p>References available on request.<br><br>Best wishes //Signe
+            </p>
+            </div>
+        
+        <script src="app.js"></script>
```

Comment:
Let's remember about `./` in path to file. Moreover, this tag should be moved to the very bottom of body container

Reply:
Hm, I don't understand this one. How can it work if I wrote the wrong path? I have changed it, and it works both ways, how can that be?